auto_gptq/utils/marlin_utils.py

```diff
@@ -142,7 +142,9 @@ def convert_to_marlin(model, model_quantlinear, quantization_config, repack: boo
         if not isinstance(module, model_quantlinear):
             continue
 
-        if module.bias is not None and torch.count_nonzero(module.bias) > 0:
+        # We could use `torch.count_nonzero(module.bias) > 0` here to discard zero bias, but this has issues when loading weights
+        # from checkpoints holding zero bias.
+        if module.bias is not None:
```

Comment:
It might be a stupid question, but why do you need this check?
`module.bias` might be either None or not.
And you can easily assign None (or not None value) to `new_module.bias`.

```python

...
new_module.bias = module.bias
...
```
?

Reply:
Right, I think it is not needed, not sure whether I wrote this & why it is not simplier in the first place.